testing/chain.go

```diff
@@ -399,9 +399,9 @@ func (chain *TestChain) GetConsensusState(clientID string, height exported.Heigh
 	return chain.App.GetIBCKeeper().ClientKeeper.GetClientConsensusState(chain.GetContext(), clientID, height)
 }
 
-// GetValsAtHeight will return the trusted validator set of the chain for the given trusted height. It will return
+// GetTrustedValidators will return the trusted validator set of the chain for the given trusted height. It will return
 // a success boolean depending on if the validator set exists or not at that height.
```

Comment:
```suggestion
// GetTrustedValidators will return the trusted validator set of the chain for the given trusted height. Otherwise
// it will return an error. 
```